File: Bpa_v2_bakc/src/main/java/com/example/Bpa_v2_bakc/entities/object_request/UserRequest.java
package com.example.Bpa_v2_bakc.entities.object_request;

import lombok.Getter;
import lombok.Setter;

@Getter
@Setter
public class UserRequest {
    private int idUser;
    private String nom;
    private String prenom;
    private String email;
    private int idRole;
    private String mot_de_passe;
    private String id_x3;
    private String interim;
    private  boolean active = true ;
}

File: Bpa_v2_front/src/app/object/userRequest.ts
import { Role } from "../models/role";

export interface UserRequest{
    idUser: number;
    nom: string;
    prenom: string;
    email: string;
    id_x3: string;
    interim: string;
    role: Role;
    mot_de_passe: string;
    active:boolean;
}